src/operators/gran_turismo.h

```diff
@@ -0,0 +1,197 @@
+/*
+    src/gran_turismo.h -- Gran Turismo 2017 tonemapping operator
+    
+    Copyright(c) 2017 by Hajime UCHIMURA @ Polyphony Digital Inc.
+    https://www.slideshare.net/nikuque/hdr-theory-and-practicce-jp
+    https://www.desmos.com/calculator/gslcdxvipg - English Translation by Romain Guy
+
+    Tone Mapper is provided under the MIT License.
+    See the LICENSE.txt file for the conditions of the license. 
+*/
+
+#pragma once
+
+#include <tonemap.h>
+
+#define glsl(x) #x
```

Comment:
This trick does not seem to be portable, or at least it doesn't compile on my version of clang on macOS.
I changed all other GLSL code in operators to the C++ multiline string literals though, that should be a less painful way (and really I should have done this from the start)

e.g. 
```C++
R"glsl(
#version 330
int main() {
    out_color = vec4(1);
}
)glsl"
```

Reply:
That's cool, I'll replace the use of the macro everywhere I use it; not just here :)